README.md

```diff
@@ -8,6 +8,8 @@ This library uses the standard Golang [SQL driver interface](https://golang.org/
 
 ### Create Connection
 
+#### With exasol dsn 
```

Comment:
Would be nice to follow the same markdown conventions across the whole documentation. I suggest using the following rules:

Use title case for section headings, preferably the APA style variant:

Capitalize the first word of any heading, including title, subtitle, subheading.
Capitalize all "major" words (nouns, verbs, adjectives, adverbs, and pronouns) in any heading, including the second part of hyphenated major words (e.g., Self-Report not Self-report).
Capitalize all words of five letters or more.
Effectively, only "minor" words of four letters or fewer, namely, conjunctions (words like and, or, nor, and but), articles (the words a, an, and the), and prepositions (words like as, at, by, for, from, in, of, on, per, to, with), are lowercased in any heading, as long as they aren't the first word.

Reply:
This is a rule we use in all public repositories our team maintaining. According to this rule, the header should look like this: `With Exasol DSN`